### ISSUE TITLE
Docker / Docker Compose support

### ISSUE THREAD

**Issue by driftywinds** [RishiBakshii/mern-chat-end-to-end-encrypted#4]
Please make arm64, amd64 docker containers for Baatchit, this would make it very easy to deploy

**Comment by RishiBakshii** [comment]
hello @driftywinds , thankyou for raising an issue here..

Coming to arm64 and amd64 docker containers are they like some kind of images? As i don't know much about them but will do my research and fulfill this issue for sure :)

**Comment by driftywinds** [comment]
> hello @driftywinds , thankyou for raising an issue here..
> 
> Coming to arm64 and amd64 docker containers are they like some kind of images? As i don't know much about them but will do my research and fulfill this issue for sure :)

yes they are images that are configured for this software called Docker that lets people deploy the app with one command. The image itself has different layers of software that are needed to run the app like next.js etc. Thank you for looking into it!
